server.js

```diff
@@ -1,16 +1,17 @@
 const express = require('express');
 const hex_sha1 = require('./sha1.js');
-const lunr = require('lunr');
 const fs = require('fs');
 const https = require('https');
 const http = require('http');
-import {ArrayFormatter, touchp, esc_query} from "./utils";
+import { ArrayFormatter, touchp, esc_query } from "./utils";
 const chokidar = require('chokidar');
 const Stream = require('stream');
 const Chain = require('stream-chain');
 const parser = require('stream-json');
 const StreamArray = require('stream-json/streamers/StreamArray');
 
+const { lunrIndexer, redisIndexer } = require('./search-index');
+const program = require('commander');
```

Comment:
... this is the other

Reply:
I didn't save this to package.json. I'm going to take it out and use environment variables instead.